devops/scripts/benchmarks/benches/umf.py

```diff
@@ -208,20 +186,56 @@ def parse_output(self, output):
             raise ValueError("Benchmark output does not contain data.")
 
         results = []
+
         for row in reader:
             try:
                 full_name = row[self.col_name]
                 pool, config = self.get_pool_and_config(full_name)
-                mean = self.get_mean(row)
-                updated_pool = self.get_preloaded_name(pool)
-                updated_config = self.get_preloaded_name(config)
+                statistics = None
+                is_row_matched_to_statistics_type = False
+
+                if not self.is_memory_overhead_checked:
```

Comment:
Please revert this condition. As a general rule, if possible, avoid negation in conditions. It's easier to read.

`if self.is_memory_overhead_checked:`